Podfile.lock

```diff
@@ -7,4 +7,6 @@ DEPENDENCIES:
 SPEC CHECKSUMS:
   yajl: 1597676af793b78d195deb0432afdf9746c0bdcd
 
-COCOAPODS: 0.39.0
+PODFILE CHECKSUM: 4bb604ae25517aab64b12b04025ea87e2eedaf32
+
+COCOAPODS: 1.0.1
```

Comment:
I have no idea how it got here - probably through the repeated rebasing onto master. It originates in a different commit https://github.com/kitt-browser/adblockplussafariios/commit/fe89753430c0bd57a51e7eb16032a93dc206f614#diff-04fcbd1c0da95c817cfec1af86aa32ad